lib/ship.rb

```diff
@@ -10,23 +10,24 @@ def initialize(name, length)
     @name = name
     @length = length
     @health = length
+    @sunk = false
   end
 
-
-
-
-
-  def length
-
-  end
-
-
   def sunk?
-
+    if @health > 0
+      @sunk = false
+    elsif @health == 0
+      @sunk = true
+    elsif @health < 0
+      @sunk = true
+    end
```

Comment:
Looks good, maybe we can refactor it once the project is complete. We can re-visit!